imagersite/imager_images/tests.py

```diff
@@ -61,3 +62,88 @@ def test_album_reference(self):
     def test_album_cover(self):
         """Test that there is a cover photo."""
         self.assertTrue(self.album.cover == self.photo)
+
+
+class AlbumView(TestCase):
+    """Test case for registration view."""
+    def setUp(self):
+        self.response = self.client.get(reverse('albums_list'))
+        self.c = Client()
+        self.user = User(username='test_user')
+        self.user.save()
+        self.photo = PhotoFactory(user=self.user)
+        self.photo.save()
+        self.album = AlbumFactory(cover=self.photo, user=self.user)
+        self.album.save()
+        self.album.photos.add(self.photo)
+        self.c.force_login(user=self.user)
+        self.logged_in_response = self.c.get(reverse('albums_list'))
+
+    def tearDown(self):
+        self.user.delete()
+        self.photo.delete()
+        self.album.delete()
+
+    def test_albums_redirect(self):
+        self.assertEquals(self.response.status_code, 302)
+
+    def test_albums_logged_in(self):
+        self.assertContains(self.logged_in_response, 'Albums', status_code=200)
+
+    def test_context_albums(self):
+        from django.db.models.query import QuerySet
+        assert type(self.logged_in_response.context['albums']) is QuerySet
+
```

Comment:
You could add tests for when there are no albums in your albums list to display on the page. Of course you'd have to add something in your template that only shows up when there are no albums to display... Same for your photos tests.